plat/renesas/rcar/platform.mk

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-PROGRAMMABLE_RESET_ADDRESS	:= 0
+PROGRAMMABLE_RESET_ADDRESS	:= 1
```

Comment:
have you tested this commit? it affects the boot flow (check bl2_el3_entrypoint.S). IIRC this was an issue